fitbenchmarking/templates/table_template.html

```diff
@@ -68,6 +68,24 @@ <h3>Errors</h3>
                         </tr>
                     {% endfor %}
                 </table>
+		{% if has_pp %}
+		<hr>
+		<section class="accordion">
```

Comment:
I like this style of hiding/showing the sections of the screen.
It might be worth doing for the table too at some point, as the profiler feels a bit hidden at the bottom of the page?

Reply:
Agreed.  I could add it here, but I feel it's not the right place for it.  Let's chat later

Reply:
I've now made this change here